benchexec/tools/dartagnan.py

```diff
@@ -26,7 +30,10 @@ def project_url(self):
     def cmdline(self, executable, options, task, rlimits):
         if task.property_file:
             options += [task.property_file]
-        return [executable] + options + list(task.input_files_or_identifier)
+        input_files = get_non_witness_input_files_or_identifier(task)
+        witness_options = ["-witness"]
+        additional_options = get_witness_options(options, task, witness_options)
```

Comment:
I can understand that this is easier to add, but it creates inconsistent tool-info modules, for example this one. The property file gets added to `options`, but then an additional `additional_options` is also created. Future developers will likely wonder why this is the case and whether it is relevant.

Can we attempt to adjust the changes to each module such that the style of the code in the module remains consistent?

Reply:
Hopefully fcba870b6534a0dea0df438d3a16903fdfbdd756 present an improvement with respect to this comment. Here I tried to keep the `witness_options` sepparate from the other options and consistent accross tool info modules.

Reply:
Hm, yes, with the additional benefit from naming it `witness_options` it seems worth it to just keep it always separate.